### ISSUE TITLE
IconButton

### ISSUE THREAD

**Issue by ericjiang97** [bumbag/bumbag-ui#35]
**Is your feature request related to a problem? Please describe.**
N/A

**Describe the solution you'd like**
A button that just holds an Icon

**Describe alternatives you've considered**

**Additional context**

**Comment by jxom** [comment]
Like this? https://bumbag.style/playroom/#?code=N4Igxg9gJgpiBcIA8AhArgFwxAdgAgAcBDAGxixgF4AdEKInAcxgCda8A3IlgSwYxohGACwgBnDO258AtCSIAjGCUEBlGDDysWEFmNoA%2BavjxIAkpHw9Lg%2Bk1bsA9EZxJH6LLhcgAvkA

**Comment by ericjiang97** [comment]
Yep, but wondering if the Button can actually fit the icon, instead of the spacing.
